scenarios/options.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+
+from useragent import Client, Server, Request
+
+
+async def server(ua):
+    invite = await ua.recv_request('INVITE')
+    await invite.respond('100 Trying')
+    await invite.respond('180 Ringing')
+    await invite.respond('200 OK')
+    await ua.recv_request('ACK')
+
+    await ua.send_request('OPTIONS')
+    await ua.recv_response('200 OK')
+
+    await asyncio.sleep(1)
+
+    await ua.send_request('BYE')
+    await ua.recv_response('200 OK')
+
+
+async def client(ua):
+    await ua.send_request('INVITE')
+    response = await ua.recv_response('200 OK', ignore=[100, 180, 183])
+    await response.ack()
+
+    bye = await ua.recv_request('BYE')
+    await bye.respond('200 OK')
+
+
+async def options():
+    async def on_recv(agent, msg):
+        if msg.method == 'OPTIONS':
+            await Request(agent, msg).respond('200 OK')
+            return True
+        return False
```

Comment:
That's a bit limiting if you ever want to do multiple responses - I'm not sure if that's ever likely?

Otherwise sure, but then I'd expand this out into more than just an `__init__` I think.